.travis.yml

```diff
@@ -13,8 +13,8 @@ matrix:
     - script: make spm_test
       env: JOB=SPM
       before_install:
+        - make bootstrap
```

Comment:
There is a test that depends on existence of `Carthage/Checkouts/Commandant`.
--
Norio Nomura @norio_nomura

2016/03/18 0:51$B!"(BJP Simard <notifications@github.com> $B$N%a%C%;!<%8(B:

> In .travis.yml:
> 
> > @@ -13,8 +13,8 @@ matrix:
> >      - script: make spm_test
> >        env: JOB=SPM
> >        before_install:
> > +        - make bootstrap
> is this still necessary?
> 
> $B!=(B
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>